Dockerfile

```diff
@@ -9,10 +9,11 @@ RUN mv dep $GOPATH/bin/dep
 RUN dep ensure --vendor-only
 
 # Build image
-RUN ./configure && make clean oauth2_proxy
+RUN CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo . && strip oauth2_proxy
```

Comment:
To get the effect of "strip" without using it, I use `-ldflags="-s -w"`. It seems like you also need `-ldflags="-X main.VERSION=${VERSION}"` (because in `version.go` it is the string "undefined").

Reply:
Thanks for pointing out.  Added new make target `build.static` and got to call it in docker build.  `-s -w` was not added there in case of need for debug information.